doc/rst/usingchapel/portability.rst

```diff
@@ -0,0 +1,48 @@
+
+
+===========
+Portability
+===========
+
+Chapel is a portable language that is designed to run on a wide variety of hardware and distributed network configurations.
+
+Operating Systems
+-----------------
+
+* Linux
+* MacOS
+* Windows (with WSL)
+
+Compute Architectures
+---------------------
+
+* CPU
+
+  * x86
+
+  * ARM
```

Comment:
I consider `x86` as covering both Intel and AMD, since it does not really matter who makes the architecture for us. If I add Intel and AMD as subcategories for `x86`, it makes sense to me to also add various vendors for `ARM`, which is much more numerious

Reply:
Oh, that's a good point.  I lazily/erroneously think of x86 as being equivalent to Intel.  My bad.

Your call, but I could definitely imagine doing something like:
* x86 (Intel, AMD)
* ARM (M1/M2, Graviton, A64FX, Raspberry Pi, …)

to get some more buzzwordy names and vendor names into the entries.  One advantage to doing so would be that if people were to Google "Chapel AMD" or "Chapel Graviton", they might land here (?).  I'm definitely open to keeping it simple as it is as well, though now that you've pointed out my mistake.

Reply:
I like that approach, I am a fan of using keywords like this to improve the SEO